src/model/resources/thermal/thermal.jl

```diff
@@ -41,3 +49,19 @@ function thermal!(EP::Model, inputs::Dict, setup::Dict)
 	EP[:eGenerationByZone] += eGenerationByThermAll
 	=# ##From main
 end
+
+function thermal_maintenance_capacity_reserve_margin_adj!(EP::Model,
+                                                          inputs::Dict)
+    dfGen = inputs["dfGen"]
+    T = inputs["T"]     # Number of time steps (hours)
+    reserves = inputs["NCapacityReserveMargin"]
```

Comment:
same comment re: calling this "reserves"

Reply:
maybe "num_capacity_reserve_margins" or something like that would work.

Reply:
resolved to `ncapres`. Since it's only very local (and the key in `inputs` is named well) I think keeping it short is fine.